.travis.yml

```diff
@@ -43,7 +43,9 @@ install:
     # Testing dependencies
     #- pip install patsy
     - pip install codecov pytest pytest-cov;
-    - pip install neuropycon_data
+    #- pip install neuropycon_data
+    #- wget -O data_nii.zip --no-check-certificate  --content-disposition https://cloud.int.univ-amu.fr/index.php/s/STtqD3Y4ZZYqpTm/download
+    #- unzip -o data_nii.zip
```

Comment:
why is this commented out? Just remove it if you don't need it any more ...

Reply:
done